src/main/java/com/revature/services/PersonLogicImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.revature.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Isolation;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.revature.domain.Person;
+import com.revature.repositories.PersonRepository;
+
+@Service
+@Transactional(readOnly=false, isolation=Isolation.READ_COMMITTED)
+public class PersonLogicImpl implements PersonLogic {
+
+	@Autowired
+	private PersonRepository personRepository;
+	
+	@Override
+	public List<Person> getPersonByFirstName(String firstName) {
+		
+		List<Person> persons = personRepository.findByFirstName(firstName);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "persons". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1488)